src/components/incidents/Incidents.js

```diff
@@ -290,7 +284,7 @@ const Incidents = () => {
   return (
     <div className="incident-reports-page">
       <div className="form-container">
-        <h1>Report Search Box</h1>
+        <h1>Search Incident Reports</h1>
```

Comment:
I think this should actually be "Filter Incident Reports" or just "Filter Reports" since it's not actually a search. 🧐

Reply:
Good catch. I updated the header to "Filter Incident Reports" so it better reflects the feature.